cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -711,7 +721,47 @@ export default {
       DropdownMenu().find(Checkbox('Custom fields')).has({ checked: false }),
     ]);
   },
-
+  verifyShowColumnsSubsection() {
+    cy.expect([
+      DropdownMenu().find(Checkbox('Holdings ID')).has({ disabled: true }),
```

Comment:
just verify that Checkbox with filter disabled: false is absent in the dropdown

Reply:
Done!